crowns.lic

```diff
@@ -117,27 +117,21 @@ class Dice
   end
 
   def check_prize
-    if @trash_items.any? { |item| Regexp.new(item) =~ right_hand }
-      bput("put my #{righthand} in bucket", 'You put', 'You drop')
-    else
-      case bput("stow #{righthand}", 'You put', 'to fit in the', 'What were', 'Stow what?')
-      when 'to fit in the'
-        message("*** Item is too big to fit in your stow container! ***")
-        exit
-      end
-      pause
-    end
-    # Check left hand as well. On lag may have items in both hands.
-    if @trash_items.any? { |item| Regexp.new(item) =~ left_hand }
-      bput("put my #{lefthand} in bucket", 'You put', 'You drop')
-    else
-      case bput("stow #{lefthand}", 'You put', 'to fit in the', 'What were', 'Stow what?')
-      when 'to fit in the'
-        message("*** Item is too big to fit in your stow container! ***")
-        exit
+    [left_hand, right_hand]
+      .compact
```

Comment:
Build list of the items to stow as prizes. If either hand is empty then `.compact` will reduce the size of the array. If no items in our hands then the array is empty and this method won't do anything and just return. No error.